### PR TITLE
DEVENGAGE-1622 Node issue

### PR DESCRIPTION
**Pull request by diemenschmaschine92**
using node version > 16 causes this error:
```
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at module.exports (/app/node_modules/webpack/lib/util/createHash.js:135:53)
    at NormalModule._initBuildHash (/app/node_modules/webpack/lib/NormalModule.js:417:16)
    at /app/node_modules/webpack/lib/NormalModule.js:452:10
    at /app/node_modules/webpack/lib/NormalModule.js:323:13
    at /app/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /app/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at context.callback (/app/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /app/node_modules/babel-loader/lib/index.js:59:103 {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```
Adding a note about version requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 ![React App flowchart](blueprint/images/flowchart.png "React app flowchart")
 
+:::primary
+ **Note**: This blueprint uses React v16. Higher versions will require additional webpack configuration.
```
Comment (jacob-shaw-genesys) [inline README.md]: @timsmithgenesys Should something about the node version be mentioned as well?

File: src/components/user-search/UserSearch.component.tsx [hunk 2]
```diff
@@ -55,7 +55,8 @@ export function UserSearch() {
                         systemPresence: ''
                     };
                     return userResult;
-                });
+                })
+                .slice(0, 5);
```
Comment (jacob-shaw-genesys) [inline src/components/user-search/UserSearch.component.tsx:59]: I encountered some intermittent 429s. The debounce in this component was supposed to fix that, but this reduces the search results and thus the API calls.

File: README.md (hunk 1 above)
Comment (timsmithgenesys) [reply README.md]: Not unless there's a specific caveat around it. However, go ahead and add a `.nvmrc` file to the `app` folder. The contents should just be the node version the project uses, e.g. `16` or `18`.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,9 @@
...
+
```
Comment (timsmithgenesys) [inline README.md:10]: Missing the closing `:::`

File: blueprint/index.md [hunk 4]
```diff
@@ -11,6 +11,9 @@ summary: |
 
 This Genesys Cloud Developer Blueprint demonstrates how to include the Genesys Cloud Javascript Platform SDK in a React app. This solution uses Typescript, although the implementation will be similar if you're using plain JavaScript. This solution includes a sample React app that explores use cases such as creating a profile page for a user and completing admin operations such as signing all the users out of a queue.
 
+:::primary
+ **Note**: This blueprint uses React v16. Higher versions will require additional webpack configuration.
+
```
Comment (timsmithgenesys) [inline blueprint/index.md:17]: missing the closing `:::`

**Review by timsmithgenesys — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jacob-shaw-genesys) [reply README.md:10]: Thanks, actually didn't know they required them haha.

**Review by timsmithgenesys — APPROVED** [review]

**Pull request merged**